lib/cylc/review_dao.py

```diff
@@ -279,17 +279,15 @@ def get_suite_job_entries(
         return (entries, of_n_entries)
 
     def is_cylc8(self, user_name, suite_name):
-        # Detemine Cylc version for a given suite: Database changes require
-        # A different database query for Cylc8.
+        """Detemine Cylc version for a given suite: Database changes require
+        A different database query for Cylc8.
+        """
         suite_info = self._db_exec(
-            user_name, suite_name, 'SELECT * FROM suite_params', []
```

Comment:
This would fail for workflows run with Cylc 8 after the bulk renaming of "suite" → "workflow"